File: app/src/main/java/intellij/class7/interfaces/tarefa/java/Subtracao.java
package intellij.class7.interfaces.tarefa.java;

public class Subtracao implements OperacaoMatematica {
    @Override
    public int calcula(int a, int b) {
        int subtracao = a - b;
        System.out.println("Subtração = " + subtracao);
        return subtracao;
    }
}

File: app/src/main/java/intellij/class7/interfaces/tarefa/kotlin/Subtracao.kt
package intellij.class7.interfaces.tarefa.kotlin

class Subtracao : OperacaoMatematica {

    override fun calcula(a: Int, b: Int): Int {
        val subtracao = a - b
        println("Subtração = $subtracao")
        return subtracao
    }
}